.github/workflows/tests.yaml

```
@@ -252,6 +253,12 @@ jobs:
         yarn run multiple_backend_test | tee /tmp/artifacts/${{ github.job }}/tests.log
       env:
         S3_LOCATION_FILE: tests/locationConfig/locationConfigTests.json
+    - name: Cleanup and upload coverage
+      uses: ./.github/actions/cleanup-and-coverage
+      with:
+        profiles: 'sproxyd mongo'
```

nit: smartish yaml way to do the same and look like it is regular yaml list :)
```suggestion
        profiles: >
            sproxyd
            mongo
```